onnx/common/ir.h

```diff
@@ -875,6 +929,11 @@ friend struct Value;
   const_graph_node_list nodes() const {
     return const_graph_node_list(output_, kNextDirection);
   }
+
+  std::vector<OpSetID>& opset_versions_mutable() {
```

Comment:
As @houseroad and I discussed, the challenge with this, as mentioned above, is that it would require significant refactoring of the modification of these members in DefaultVersionConverter.  If it would be particularly important, I can do this.

Reply:
Yeah, add a note in the comment here, let's do it in the following PR.